spec/features/view_list_of_authors_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.feature "View list of Authors", type: :feature do
+  scenario "with some authors" do
+    create(:author, name: "Martin Fowler")
+    create(:author, name: "Uncle Bob")
+
+    visit authors_path
+
+    expect(page).to have_author("Martin Fowler")
+    expect(page).to have_author("Uncle Bob")
+  end
+
+  scenario "without authors" do
```

Comment:
Do you think this might already be sufficiently covered by the view spec? The routing and controller action are the ame as already exercised by the scenario above.